recipes/default.rb

```diff
@@ -21,7 +21,7 @@
 
 # Start+Enable the lvmetad service on RHEL7, it is required by default
 # but not automatically started
-if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7
+if node['platform_family'] == 'rhel' && node[platform] != 'amazon' && node['platform_version'].to_i >= 7
```

Comment:
You'll need some more complex logic here actually because on Chef <= 12 Amazon is platform_family of RHEL, but on 13+ it's amazon.